tests/testthat/test-pg.R

```diff
@@ -1,3 +1,11 @@
+# If you get an error like:
+# ```
+# psql: error: connection to server on socket "/tmp/.s.PGSQL.5432" failed:
+# No such file or directory
+# Is the server running locally and accepting connections on that socket?
+# ```
+# Check the data-analysis-book section 9.5 for instructions on how to fix it.
```

Comment:
Since this is a public facing package we should explain what is happening/how to fix it instead of an internal resource. So we can say this is caused from not having a psql local server set up, for the tests to pass you must set up a local psql server.